secrets-provider/azure/README.md

```diff
@@ -25,7 +27,7 @@ az keyvault create -l westus -n pulumi --resource-group $RESOURCE_GROUP_NAME
 az keyvault key create --name pulumi-secret --vault-name pulumi
 
 # Finally, set the relevant permissions on the keyvault
-az keyvault set-policy --name pulumi --object-id $YOUR_OBJECT_ID --key-permissions decrypt get create delete list update import backup restore recover
+az keyvault set-policy --name pulumi --object-id $YOUR_OBJECT_ID --key-permissions decrypt get create delete list update import backup restore recover encrypt
```

Comment:
Without the `encrypt` permission set here, you'll get 403 errors back during `pulumi init`